http/users/http.go

```diff
@@ -0,0 +1,173 @@
+package users
+
+import (
+	"encoding/json"
+	"fmt"
+	"layer/user/models"
+	"layer/user/services"
+	"net/http"
+	"reflect"
+	"strconv"
+
+	"github.com/gorilla/mux"
+)
+
+type Handler struct {
+	hndlr services.User
+}
+
+func New(s services.User) Handler {
+	return Handler{s}
+}
+
+func (srv Handler) GetUserByIdHandler(res http.ResponseWriter, req *http.Request) {
+	res.Header().Set("Content-type", "application/json")
+
+	params := mux.Vars(req)
+
+	userId := params["id"]
+
+	id, err := strconv.Atoi(userId)
```

Comment:
check for invalid id, id should be greater than 0.

Reply:
will do it by today EOD.